### PR TITLE
Assign admin role to Cliniko users

### PR DESCRIPTION
**Pull request by lionsdwarf**
When this gets deployed, let's make sure we update the code in the Auth0 Rule and de-activate/delete the to-be-deprecated Authorization Extension.

### CODE REVIEW

File: apps/api/models/user/mapper.js [hunk 1]
```diff
@@ -22,8 +23,17 @@ export const findAll = (
 
 export const create = email => auth0.createUser(toAuth0(email))
 
+export const assignRole = (userId, role) =>
+  auth0.assignRolestoUser({ id: userId }, { roles: [role.id] })
+
 export const createUsers = emails => Promise.all(emails.map(create))
 
+export const assignAdminRoles = async users => {
```
Comment (craigspaeth) [inline apps/api/models/user/mapper.js]: It'll be annoying to mock `auth0` but mind adding an integration test to this function? Then this LGTM 👍

**Comment by craigspaeth** [comment]
Feel free to merge after addressing my test comment. Thanks!

File: apps/api/models/user/mapper.js (hunk 1 above)
Comment (lionsdwarf) [reply apps/api/models/user/mapper.js]: Np, will do.

**Comment by lionsdwarf** [comment]
FYI, I added and commented-out the Rule code in Auth0 prod, so flipping the switch will only require uncommenting and deleting the existing role assignment, and then deleting the Authorization integration and Rule.

**Pull request merged**